spring-boot-modules/spring-boot-crud/src/main/java/com/baeldung/entity_dto_differences/Application.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.entity_dto_differences;
```

Comment:
Java package names should not have underscores in them. We follow these guidelines for naming:
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names